src/utils/genres.ts

```diff
@@ -0,0 +1,116 @@
+export enum Genre {
+  ELECTRONIC = 'Electronic',
+  ROCK = 'Rock',
+  METAL = 'Metal',
+  ALTERNATIVE = 'Alternative',
+  HIP_HOP_RAP = 'Hip-Hop/Rap',
+  EXPERIMENTAL = 'Experimental',
+  PUNK = 'Punk',
+  FOLK = 'Folk',
+  POP = 'Pop',
+  AMBIENT = 'Ambient',
+  SOUNDTRACK = 'Soundtrack',
+  WORLD = 'World',
+  JAZZ = 'Jazz',
+  ACOUSTIC = 'Acoustic',
+  FUNK = 'Funk',
+  R_AND_B_SOUL = 'R&B/Soul',
+  DEVOTIONAL = 'Devotional',
+  CLASSICAL = 'Classical',
+  REGGAE = 'Reggae',
+  PODCASTS = 'Podcasts',
+  COUNTRY = 'Country',
+  SPOKEN_WORK = 'Spoken Word',
+  COMEDY = 'Comedy',
+  BLUES = 'Blues',
+  KIDS = 'Kids',
+  AUDIOBOOKS = 'Audiobooks',
+  LATIN = 'Latin',
+
+  // Electronic Subgenres
+  TECHNO = 'Techno',
+  TRAP = 'Trap',
+  HOUSE = 'House',
+  TECH_HOUSE = 'Tech House',
+  DEEP_HOUSE = 'Deep House',
+  DISCO = 'Disco',
+  ELECTRO = 'Electro',
+  JUNGLE = 'Jungle',
+  PROGRESSIVE_HOUSE = 'Progressive House',
+  HARDSTYLE = 'Hardstyle',
+  GLITCH_HTOP = 'Glitch Hop',
+  TRANCE = 'Trance',
+  FUTURE_BASE = 'Future Bass',
+  FUTURE_HOUSE = 'Future House',
+  TROPICAL_HOUSE = 'Tropical House',
+  DOWNTEMPO = 'Downtempo',
+  DRUM_AND_BASS = 'Drum & Bass',
+  DUBSTEP = 'Dubstep',
+  JERSEY_CLUB = 'Jersey Club',
+  VAPORWARE = 'Vaporwave',
+  MOOMBAHTON = 'Moombahton'
+}
+
+export const ELECTRONIC_PREFIX = 'Electronic - '
+
+export const ELECTRONIC_SUBGENRES: { [key in Genre]?: string } = {
+  [Genre.TECHNO]: `${ELECTRONIC_PREFIX}Techno`,
```

Comment:
nit: maybe use `${ELECTRONIC_PREFIX}${Genre.TECHNO}` to avoid spelling errors

Reply:
great